spring-cloud-services-spring-connector/src/main/java/io/pivotal/spring/cloud/service/config/PlainTextConfigClientAutoConfiguration.java

```diff
@@ -30,15 +31,19 @@
  * available in the container.
  *
  * @author Daniel Lavoie
+ * @author Roy Clarkson
  */
 @Configuration
 @ConditionalOnClass({ OAuth2ProtectedResourceDetails.class,
 		ConfigClientProperties.class })
+@EnableConfigurationProperties({ ConfigClientOAuth2ResourceDetails.class,
+		ConfigClientProperties.class })
 public class PlainTextConfigClientAutoConfiguration {
 
 	@Bean
-	@ConditionalOnBean(ConfigClientOAuth2ResourceDetails.class)
```

Comment:
Just occurred to me that being condition on `ConfigClientOAuth2ResourceDetails` (as it was prior to this PR) is kinda silly if the bean method takes `ConfigClientOAuth2ResourceDetails` as a parameter. If the bean didn't exist, then it'd fail regardless of the `@ConditionalOnBean` annotation. But being more specific on given properties is worthwhile. (Nothing to see here...just commenting that what's here is better than what was there before.)